From 06e33ac0942e410493d84cd7159db3238d6d87e3 Mon Sep 17 00:00:00 2001
From: Yang Bin <yangbin@runtime-server1.sh.intel.com>
Date: Sat, 28 Dec 2013 18:38:57 +0800
Subject: Modify Android.mk to fix build error.

Change-Id: I20274f4b3b647999e4f97fbb70b6c036d8d510b6
Signed-off-by: Yang Bin <yangbin@runtime-server1.sh.intel.com>
---
 Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Android.mk b/Android.mk
index 0a1fce1..3c93023 100644
--- a/Android.mk
+++ b/Android.mk
@@ -115,7 +115,7 @@ endif
 
 ifeq (,$(wildcard frameworks/base/core/jni/android_hardware_Camera.h))
 LOCAL_C_INCLUDES += \
-	$(TARGET_OUT_HEADERS)/camera_extension
+	vendor/intel/hardware/camera_extension/include/
 endif
 
 LOCAL_C_FLAGS =+ -fno-pic
-- 
1.7.4.1

